lldb/utils/lit-cpuid/lit-cpuid.cpp

```diff
@@ -22,13 +22,13 @@ using namespace llvm;
 int main(int argc, char **argv) {
 #if defined(__i386__) || defined(_M_IX86) || \
     defined(__x86_64__) || defined(_M_X64)
-  if (std::optional<StringMap<bool>> features =
+  if (const std::optional<StringMap<bool>> features =
           sys::getHostCPUFeatures(features)) {
-    if ((*features)["sse"])
+    if (features->contains("sse"))
```

Comment:
`contains` will only check if the key exists in the map, but you want to actually get the `bool` if it exists and default to `false` if it doesn't. I think `lookup` fits better.

Reply:
Doh, of course.

Reply:
Fixed.